gizmo.py

```diff
@@ -0,0 +1,55 @@
+class Gizmo:
+    def __init__(self, name):
+        self.name = name
+
+    def speak(self):
+        print (self.name)
+
+def hello(name, country = "finland"):
+    print("Hello ",name,", how are things in ",country,"?")
+
+def spell(woord):
+    teller = 1
+    for i in woord:
```

Comment:
```suggestion
    for letter in woord:
```
Misschien duidelijker?